lib/inspec/resources/opa_api.rb

```diff
@@ -18,6 +24,10 @@ def allow
       @content["result"]
     end
 
+    def resource_id
+      @url
```

Comment:
This might lead to nil error since in line 16 @url is initialized as `@url = opts[:url] || nil`.
Suggested fix: `@nil || "OPA api"

Reply:
Yes but there is validation also below that which does not allow URL as nil.